CHANGELOG-1.3.md

```diff
@@ -6,3 +6,28 @@ in 1.3.x patch versions.
 
 1.3.1 (????-??-??)
 ------------------
+
+1.3.0 (2016-03-19)
```

Comment:
Would it make sense to remove "1.3.1 (????-??-??)" entirely if we're now on 1.4.0?